docs/README.md

```diff
@@ -1,7 +1,13 @@
 # Documentation
 
-To generate HTML documentation run the `generate_docs.py` script from any sub-dir of the
-repository (most likely `build`) or enable and use build target 'docs' (see details below).
+To generate HTML documentation, run the `generate_docs.py` script from any sub-directory of the repository (most likely `build`).
+To display the proper version of UMF in the documentation title, set the `UMF_VERSION` variable before running the script.
+
+```bash
+$ UMF_VERSION=<your version, e.g. "0.12.1"> python ./docs/generate_docs.py
```

Comment:
heh, just a few lines above we say it has to be run from any sub-dir, and here you suggest it can be run from the root dir of the repo - pls update the example :)

Reply:
done